customer-service/channel-integration-framework/TwilioSampleInteg_version2/wwwroot/js/site.js

```diff
@@ -779,10 +789,45 @@ function placeCall() {
     var params = {
         To: "+" + document.getElementById('dialerPhoneNumber').value
     };
+    phone.state = PhoneState.Idle;
     log('Placing a call to ' + params.To + '...');
     phone.name = params.To;
 
     Twilio.Device.connect(params);
+    phone.state = PhoneState.Dialing;
+    Microsoft.CIFramework.setPresence("onVoiceCall").then(
+        function (result) {
+            if (!result)
+                document.getElementById("setPresenceText").innerHTML = "OC Presence is in error state";
```

Comment:
Will this string be shown to customer? if so, did you get the strings scrubbed ?

Reply:
yes